src/bin/flamegraph.rs

```diff
@@ -131,6 +133,18 @@ struct Opt {
     )]
     fontwidth: f64,
 
+    /// Color of UI text such as the search and reset zoom buttons
+    #[clap(
+        long = "uicolor",
+        default_value = defaults::UI_COLOR,
+        value_parser = |s: &str| {
+            parse_hex_color(s)
+                .ok_or_else(|| format!("Expected a color in hexadecimal format, got: {}", s))
+        },
+        value_name = "STRING"
```

Comment:
Feels weird to specify `value_name` as `STRING`. It means that the help text will read:
```
--uicolor STRING
```
which doesn't seem particularly helpful. How about setting it to something like `COLOR` or `#RRGGBB`?

Reply:
Ah, I thought that was some sort of type checking thing, changing it to #RRGGBB